coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/RelToTrinoConverter.java

```diff
@@ -277,6 +284,37 @@ public Result visit(Correlate e) {
   public Context aliasContext(Map<String, RelDataType> aliases, boolean qualified) {
     // easier to keep inner class for accessing 'aliases' and 'qualified' variables as closure
     return new AliasContext(TrinoSqlDialect.INSTANCE, aliases, qualified) {
+      /**
+       * This function is overriden to handle the conversion of {@Link org.apache.calcite.rex.RexFieldAccess} like `F(X1..Xn).Y`
+       * where `F` is a Coral function which is required to be converted into a Trino function by a specific SqlCallTransformer.
+       * {@link com.linkedin.coral.common.transformers.SqlCallTransformer} only converts the function
+       * defined in a SqlCall, however, RexFieldAccess is converted into a SqlIdentifier by default
+       * in Calcite referring to {@link org.apache.calcite.rel.rel2sql.SqlImplementor.Context#toSql(RexProgram, RexNode)}).
+       * Therefore RexFieldAccess is converted into a SqlCall instead in this function.
+       * @param program Required only if {@code rex} contains {@link RexLocalRef}
+       * @param rex Expression to convert
+       * @return SqlNode
+       */
+      @Override
+      public SqlNode toSql(RexProgram program, RexNode rex) {
```

Comment:
I feel that this overriding should be added to `CoralRelToSqlNodeConverter` in coral-common which #351 does, because:
1. It needs to apply for both Coral-Spark and Coral-Trino
2. `RelToTrinoConverter` will be deprecated and removed soon after we finish transformer migration for Coral-Trino.